docker-compose.yml

```diff
@@ -0,0 +1,88 @@
+version: '3'
+
+volumes:
+    prometheus_data: {}
+
+services:
+  ct-test-srv:
+    build:
+      context: .
+      dockerfile: docker/ct-test-srv/Dockerfile
+    command: ct-test-srv -config /test/config/cttestsrv.json
+    ports:
```

Comment:
This will break the usage pattern I described in the PR description:
> You can break certificate submission for log-two by running:
curl -X POST -d '{"path":"/ct/v1/add-chain","code":404,"response":{"error":"oh noes!"}}' localhost:4601/add-mock

There's no test component that interacts with the CT log "mock" API. Eventually it would be nice to write a `.go` file that runs in a container, interacts with the CT log mock API and then interrogates the Alertmanager API to ensure the correct alerts fired but until then I think we need to expose the CT log ports for manual poking/prodding.